src/views/Login.vue

```diff
@@ -102,12 +115,24 @@ export default class Login extends Vue {
   valid = true
   error = false
   loading = false
+  source = 'moonraker'
+  availableSources = [this.source]
+
+  mounted () {
+    httpClient.get('/access/info', { withAuth: false })
+      .then(r => {
+        this.source = r.data.result.default_source
+        this.availableSources = r.data.result.available_sources
+      }).catch(() => {
+        // ignore
+      })
+  }
```

Comment:
Maybe move this to `auth.api.ts`, I thinlk it would make much more sense there!

Reply:
Moved it to the auth store actions, seemed more fitting to avoid multiple imports of the auth API